### PR TITLE
Add fixtures for testing phases 1 and 2 of Dockerfiles

### PR DESCRIPTION
**Pull request by natalieparellano**

### CODE REVIEW

**Comment by natalieparellano** [comment]
This should probably be blocked on https://github.com/buildpacks/pack/pull/1478 as CI won't pass until pack has been updated to support builder creation with extensions

**Comment by AidanDelaney** [comment]
Looks like we need to release a `pack` with support for `Dockerfiles` before these samples changes can land.

**Comment by natalieparellano** [comment]
@AidanDelaney the pipeline is green! WDYT?

File: Makefile
Comment (natalieparellano) [inline Makefile:1]: I have to confess, I'm not sure why this is needed.

File: Makefile
Comment (natalieparellano) [inline Makefile:41]: Only changed the pull policy for alpine, as that's the only builder with extensions.

Comment (AidanDelaney) [reply Makefile:41]: AFAIK it stops `pack` trying to look for `cnbs/sample-builder` on a registry.

**Review by AidanDelaney — APPROVED** [review]
PR provides an example where the image is switched to `run-image-curl`.  Also demonstrates `generate` installing the a `tree` binary via `apk`.  LGTM.

**Pull request merged**
